pkg/kubernetes/kubernetes.go

```diff
@@ -28,7 +27,7 @@ func Init() error {
 		return fmt.Errorf("can't connect to a Kubernetes cluster: %v", err)
 	}
 
-	version, err := pkg_version.GetLatestRelease(standalone.DaprGitHubOrg, standalone.DaprGitHubRepo)
+	version, err := pkg_version.GetLatestRelease(pkg_version.DaprGitHubOrg, pkg_version.DaprGitHubRepo)
```

Comment:
nit: pkg_version seems weird naming :) I know someone renamed it. isn't `cli_ver` more readable ? how do you think?

Reply:
I agree with that, changed it.